lib/code_corps_web/controllers/organization_invite_controller.ex

```diff
@@ -1,34 +1,53 @@
 defmodule CodeCorpsWeb.OrganizationInviteController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
-  import CodeCorps.Helpers.Query, only: [id_filter: 2]
-  alias CodeCorps.{Emails, Mailer, OrganizationInvite}
+  alias CodeCorps.{Emails, Helpers.Query, Mailer, OrganizationInvite, User}
 
-  plug :load_and_authorize_resource, model: OrganizationInvite, only: [:create, :update]
-  plug JaResource
+  action_fallback CodeCorpsWeb.FallbackController
+  plug CodeCorpsWeb.Plug.DataToAttributes
+  plug CodeCorpsWeb.Plug.IdsToIntegers
 
-  @spec model :: module
-  def model, do: CodeCorps.OrganizationInvite
+  @spec index(Conn.t, map) :: Conn.t
+  def index(%Conn{} = conn, %{} = params) do
+    with organization_invites <- OrganizationInvite |> Query.id_filter(params) |> Repo.all do
+      conn |> render("index.json-api", data: organization_invites)
+    end
+  end
 
-  def filter(_conn, query, "id", id_list) do
-    query |> id_filter(id_list)
+  @spec show(Conn.t, map) :: Conn.t
+  def show(%Conn{} = conn, %{"id" => id}) do
+    with %OrganizationInvite{} = organization_invite <- OrganizationInvite |> Repo.get(id) do
+      conn |> render("show.json-api", data: organization_invite)
+    end
   end
 
-  def handle_create(_conn, attributes) do
-    OrganizationInvite.create_changeset(%OrganizationInvite{}, attributes)
+  @spec create(Plug.Conn.t, map) :: Conn.t
+  def create(%Conn{} = conn, %{} = params) do
+    with %User{} = current_user <- conn |> Guardian.Plug.current_resource,
+         {:ok, :authorized} <- current_user |> Policy.authorize(:create, %OrganizationInvite{}, params),
+         {:ok, %OrganizationInvite{} = organization_invite} <- %OrganizationInvite{} |> OrganizationInvite.create_changeset(params) |> Repo.insert do
+
+      send_email(organization_invite)
```

Comment:
There was a merge conflict due to a recent merge. The rebase made the changeset really weird so I had to come in and do this manually. You'll see below that `render_create/2` was removed, but I kept the private `send_email/1` and opted not to handle it within the `with` but inside the result, where I think the async process makes more sense.